docs/docsite/rst/porting_guides.rst

```diff
@@ -0,0 +1,12 @@
+.. _porting_guides:
+
+**********************
+Ansible Porting Guides
+**********************
+
+This section links to the various porting guides that help you upgrade your playbooks and modules from one version of Ansible to the next. Although this isn't a complete list it should assist you. I you believe any extra information would be useful in these pages you can edit by clicking ` Edit on GitHub` on the top right, or raising an issue
```

Comment:
"This section lists porting guides that can help you upgrade your playbooks and modules from one version of Ansible to the next. Please note that this is not a complete list. If you believe any extra information would be useful in these pages, you can edit by clicking ` Edit on GitHub` on the top right, or raising an issue."